### DISCUSSION TITLE
最近会出现频繁重启

### DISCUSSION THREAD

**Discussion by DrCaptainJA** [LmeSzinc/AzurLaneAutoScript#3432] [Q&A]
会出现频繁Get_Ship然后引发报错然后重启
以下是一次报错（还有个第一次看到的别的按键报错一起触发了：
WITHDRAW button found, wait until map loaded to prevent    
         bugs in game client                                                       
INFO     12:03:44.610 │ Function calls:                                            
                   <string>   L1 <module>                                          
                   spawn.py L105 spawn_main()                                      
                   spawn.py L118 _main()                                           
                 process.py L297 _bootstrap()                                      
                 process.py  L99 run()                                             
         process_manager.py L151 run_process()                                     
                    alas.py L514 loop()                                            
                    alas.py  L68 run()                                             
                    alas.py L245 exercise()                                        
                exercise.py L201 run()                                             
                      ui.py L250 ui_ensure()                                       
                      ui.py L228 ui_goto()                                         
                      ui.py L478 ui_additional()                                   
                    base.py L129 appear_then_click()                               
                 control.py  L35 click()                                           
                  device.py L183 handle_control_check()                            
                  device.py L225 click_record_check()                              
                  device.py  L41 show_function_call()                              
WARNING  12:03:44.614 │ Too many click between 2 buttons: GET_SHIP, WITHDRAW       
WARNING  12:03:44.615 │ History click: ['GET_SHIP', 'MAIN_GOTO_CAMPAIGN',          
         'WITHDRAW', 'GET_SHIP', 'WITHDRAW', 'GET_SHIP', 'WITHDRAW', 'GET_SHIP',   
         'WITHDRAW', 'GET_SHIP', 'WITHDRAW', 'GET_SHIP', 'WITHDRAW']               
ERROR    12:03:44.616 │ GameTooManyClickError: Too many click between 2 buttons:   
         GET_SHIP, WITHDRAW                                                        
WARNING  12:03:44.617 │ Saving error: ./log/error/1705118624617                    
WARNING  12:03:44.801 │ Game stuck, com.bilibili.azurlane will be restarted in 10  
         seconds                                                                   
WARNING  12:03:44.802 │ If you are playing by hand, please stop Alas

**Comment by LostMyMind** [comment]
我感觉你需要提个issues，顺带把完整的log也带上，在这发没有issues关注的人多。

**Comment by DrCaptainJA** [comment]
好滴感谢&nbsp;


萨达
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年1月19日(星期五) 晚上9:14
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [LmeSzinc/AzurLaneAutoScript] 最近会出现频繁重启 (Discussion #3432)





 
我感觉你需要提个issues，顺带把完整的log也带上，在这发没有issues关注的人多。
